docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
```

Comment:
I would suggest moving this file to `test` directory, as it's used mostly for tests and this could help indicate that. 
It would also be great to add some info to ready on this topic (how to easly run tests using this).

Reply:
I was thinking about `test/docker` dir